components/director/internal/domain/bundlereferences/service.go

```diff
@@ -20,6 +20,7 @@ type BundleReferenceRepository interface {
 	ListByBundleIDs(ctx context.Context, objectType model.BundleReferenceObjectType, tenantID string, bundleIDs []string, pageSize int, cursor string) ([]*model.BundleReference, map[string]int, error)
 }
 
+// UIDService missing godoc
```

Comment:
Consider adding real go docs as part of linter issues fixing. This is a default value and it is not intended to be copied adding more and more technical debt. Its actual intent was to trigger a boy scout principal feeling in people and make them fix a few comments around the place where the task is involved instead of adding more of them.